functions-runtime/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM denoland/deno:1.29.1 as production
```

Comment:
what's the difference?

Reply:
I changed the base image, since the `alpine-1.29.1` image has less vulnerabilities than `1.29.1`.